src/platform-aws.cpp

```diff
@@ -533,21 +536,7 @@ int platform_init(const char **provider_filter)
 	 * The NVLSTree chunk size can not be larger than the NVLS chunk size,
 	 * so we ensure both are set to 512KiB.
 	 */
-	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET, "Setting NCCL_NVLSTREE_MAX_CHUNKSIZE to 512KiB");
-	ret = setenv("NCCL_NVLSTREE_MAX_CHUNKSIZE", "524288", 0);
-	if (ret != 0) {
-		NCCL_OFI_WARN("Unable to set NCCL_NVLSTREE_MAX_CHUNKSIZE");
-		ret = -errno;
-		goto exit;
-	}
-
-	NCCL_OFI_INFO(NCCL_INIT | NCCL_NET, "Setting NCCL_NVLS_CHUNKSIZE to 512KiB");
-	ret = setenv("NCCL_NVLS_CHUNKSIZE", "524288", 0);
-	if (ret != 0) {
-		NCCL_OFI_WARN("Unable to set NCCL_NVLS_CHUNKSIZE");
-		ret = -errno;
-		goto exit;
-	}
+	env_manager::getInstance().insert_envvar("NCCL_NVLSTREE_MAX_CHUNKSIZE", "524288", false);
```

Comment:
Is there a reason why here we keep setting `NCCL_NVLSTREE_MAX_CHUNKSIZE` but removed setting `NCCL_NVLS_CHUNKSIZE`?

Reply:
Mistake.  Fixed.